### PR TITLE
Added fetching backup size of volume backup

### PR DESCRIPTION
**Pull request by prashanthpx**
**What type of PR is this?**
Ehancement

**What this PR does / why we need it**:
As part of px-backup, we need to fetch backup size fort every volume backed. This PR fetches the same from the respective cloud provider and populates in application backup CR.

**Does this PR change a user-facing CRD or CLI?**:
No

**Is a release note needed?**:
No

**Does this change need to be cherry-picked to a release branch?**:
2.5

### CODE REVIEW

File: pkg/applicationmanager/controllers/applicationbackup.go [hunk 1]
```diff
@@ -821,7 +821,10 @@ func (a *ApplicationBackupController) backupResources(
 	if err = a.client.Update(context.TODO(), backup); err != nil {
 		return err
 	}
-
+	// Only on success compute the total backup size
```
Comment (disrani-px) [inline pkg/applicationmanager/controllers/applicationbackup.go:824]: Needs to be before the call to Update() above

File: pkg/apis/stork/v1alpha1/applicationbackup.go [hunk 2]
```diff
@@ -57,6 +57,7 @@ type ApplicationBackupStatus struct {
 	TriggerTimestamp    metav1.Time                      `json:"triggerTimestamp"`
 	LastUpdateTimestamp metav1.Time                      `json:"lastUpdateTimestamp"`
 	FinishTimestamp     metav1.Time                      `json:"finishTimestamp"`
+	BackupSize          uint64                           `json:"backupSize"`
```
Comment (disrani-px) [inline pkg/apis/stork/v1alpha1/applicationbackup.go:60]:
```suggestion
	Size          uint64                           `json:"size"`
```

File: pkg/applicationmanager/controllers/applicationbackup.go (hunk 1 above)
Comment (prashanthpx) [reply pkg/applicationmanager/controllers/applicationbackup.go:824]: Fixed.

File: pkg/apis/stork/v1alpha1/applicationbackup.go (hunk 2 above)
Comment (prashanthpx) [reply pkg/apis/stork/v1alpha1/applicationbackup.go:60]: Fixed.

**Comment by prashanthpx** [comment]
@disrani-px - Review comments addressed.

File: pkg/apis/stork/v1alpha1/applicationbackup.go [hunk 3]
```diff
@@ -57,6 +57,7 @@ type ApplicationBackupStatus struct {
 	TriggerTimestamp    metav1.Time                      `json:"triggerTimestamp"`
 	LastUpdateTimestamp metav1.Time                      `json:"lastUpdateTimestamp"`
 	FinishTimestamp     metav1.Time                      `json:"finishTimestamp"`
+	BackupSize          uint64                           `json:"size"`
```
Comment (disrani-px) [inline pkg/apis/stork/v1alpha1/applicationbackup.go:60]:
```suggestion
	Size          uint64                           `json:"size"`
```

Comment (prashanthpx) [reply pkg/apis/stork/v1alpha1/applicationbackup.go:60]: Fixed.

**Pull request merged**